devel/folly/Portfile

```diff
@@ -16,11 +16,9 @@ maintainers         nomaintainer
 description         An open-source C++ library developed and used at Facebook.
 long_description    ${description}
 
-checksums           rmd160  8ab6f6652a14a34610b27ca97b70476400f9defe \
-                    sha256  73c5bfa832ff9e1053dd3c3010850280da111f7e75b36b6c86cc791e22a684f9 \
-                    size    2318525
-
-patchfiles          patch-cmakelists.diff patch-cmake-config.diff
```

Comment:
Since this patchfile is no longer being used, it should be deleted.

Reply:
Done.

Reply:
patchfiles plural. Thank you.